symmetric-db/src/main/java/org/jumpmind/db/platform/mssql/MsSql2005DdlBuilder.java

```diff
@@ -87,13 +88,24 @@ protected void dropColumnChangeDefaults(Table sourceTable, Column sourceColumn,
 
     @Override
     public String getSqlType(Column column) {
+        Map<String, String> env = System.getenv();
+        boolean useVarcharForText = "y".equalsIgnoreCase(env.get("USE_VARCHAR_FOR_TEXT"));
```

Comment:
This should be a system property using our dot naming convention instead of an environment variable.  For example, use System.getProperty("mssql.use.varchar.for.lob", "true").equalsIgnoreCase("true").

Reply:
I have replaced it by getting the property as you suggested.